### ISSUE TITLE
Attributes to images

### ISSUE THREAD

**Issue by oscarotero** [arve0/markdown-it-attrs#46]
Hi, I'd like to add some atributes to images but it doesn't work. Not sure whether is a `markdown-it-attrs` issue or I'm doing something wrong.
Thanks!

Example input:
```md
![Alt text](image.jpg){.image}
```

Current output:
```html
<p class="image"><img src="image.jpg" alt="Alt text"></p>
```

Expected output:
```html
<p><img src="image.jpg" alt="Alt text" class="image"></p>
```

**Comment by arve0** [comment]
Thanks for reporting. Seems to be a bug. I'll look into it.

**Comment by arve0** [comment]
Fixed in `v1.1.0`.

**Comment by oscarotero** [comment]
Thank you so much!
